sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic qmake-utils
+
+DESCRIPTION="GUI for personalization of Yubico's YubiKey"
+SRC_URI="https://github.com/Yubico/yubikey-personalization-gui/archive/${P}.tar.gz"
+HOMEPAGE="https://github.com/Yubico/yubikey-personalization-gui"
+
+KEYWORDS="~amd64"
+SLOT="0"
+LICENSE="BSD-2"
+IUSE="debug test"
+
+RDEPEND="
+	>=sys-auth/ykpers-1.14.0
+	>=sys-auth/libyubikey-1.6
+	dev-qt/qtcore:5
+	dev-qt/qtwidgets:5
+	dev-libs/glib:2
+	virtual/libusb:1"
+DEPEND="${RDEPEND}
+	test? ( dev-qt/qttest:5 )
+	virtual/pkgconfig"
+
+DOCS=( NEWS README )
```

Comment:
Please check if this is even needed, those two files would be installed by einstalldocs automatically.

Reply:
It doesn't seem to be picking those files with or without DOCS, removing the line.